tests/FakerDotNet.Tests/Fakers/RickAndMortyFakerTests.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using FakeItEasy;
+using FakerDotNet.Data;
+using FakerDotNet.Fakers;
+using NUnit.Framework;
+
+namespace FakerDotNet.Tests.Fakers
+{
+    [TestFixture]
+    [Parallelizable]
+    public class RickAndMortyFakerTests
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            _fakerContainer = A.Fake<IFakerContainer>();
+            _rickAndMortyFaker = new RickAndMortyFaker(_fakerContainer);
+        }
+
+        private IFakerContainer _fakerContainer;
+        private IRickAndMortyFaker _rickAndMortyFaker;
+
+        [Test]
+        public void Character_returns_a_character()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(RickAndMortyData.Characters)))
```

Comment:
With the change to the data class I mentioned above it will remove the need to use the IsSameSequenceAs method.